src/main/java/org/dave/compactmachines3/gui/framework/widgets/WidgetPanel.java

```diff
@@ -21,7 +21,7 @@ public WidgetPanel() {
         // Also notify widgets when the mouse entered or exited their area
         this.addListener(MouseMoveEvent.class, (event, widget)-> {
 
-            //Logz.info("[%s] Moved mouse to x=%d, y=%d", widget.id, innerX, innerY);
+            //Logz.info("[{}] Moved mouse to x={}, y={}", widget.id, innerX, innerY);
```

Comment:
Prefer full removal of unnecessary logging

Reply:
Noted